### ISSUE TITLE
return binding.rmdir(pathModule._makeLong(path));

### ISSUE THREAD

**Issue by MeijeSibbel** [Storj/core#463]
### Package Versions

Replace the values below using the output from `npm list storj`. Use ```npm list -g storj``` if installed globally.

```
storj-bridge@0.6.5 /home/gordon/Code/storj-bridge
└── storj@0.6.5  -> /home/gordon/Code/storj-core

storj -> v4.0.0-rc.5 (hotfix)
```

Replace the values below using the output from `node --version`.

```
v4.5.0
```

### Expected Behavior

```
Upload proceeds. 
```

### Actual Behavior

```
 storj -k meije0612 -c 1 upload-file 57af3a1e8f266a7d4e391055 D:\20GB
 TIMEOUT 2
)
18
 [Tue Sep 27 2016 10:33:30 GMT+0200 (W. Europe Daylight Time)] [warn]   A redundancy of 0 means files will not be mirrored!
 [Tue Sep 27 2016 10:33:30 GMT+0200 (W. Europe Daylight Time)] [info]   1 file(s) to upload.
 [Tue Sep 27 2016 10:33:32 GMT+0200 (W. Europe Daylight Time)] [info]   Generating encryption key...
 [Tue Sep 27 2016 10:33:32 GMT+0200 (W. Europe Daylight Time)] [info]   Encrypting file "D:/20GB"
 [Tue Sep 27 2016 11:00:15 GMT+0200 (W. Europe Daylight Time)] [info]   [ 20GB ] Encryption complete
 [Tue Sep 27 2016 11:00:15 GMT+0200 (W. Europe Daylight Time)] [info]   [ 20GB ] Creating storage token... (retry: 0)
 [Tue Sep 27 2016 11:00:18 GMT+0200 (W. Europe Daylight Time)] [info]   [ 20GB ] Creating storage token... (retry: 1)
 [Tue Sep 27 2016 11:00:20 GMT+0200 (W. Europe Daylight Time)] [info]   [ 20GB ] Storing file, hang tight!
 [Tue Sep 27 2016 11:00:20 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard D:\temp\cd100aeb32a9 index 0
 [Tue Sep 27 2016 11:01:11 GMT+0200 (W. Europe Daylight Time)] [info]   Hash for this shard is: fa4a3f79cd010f82d4697d5e7c3e00f03881eba9
 [Tue Sep 27 2016 11:02:52 GMT+0200 (W. Europe Daylight Time)] [info]   Audit generation for shard done.
 [Tue Sep 27 2016 11:02:52 GMT+0200 (W. Europe Daylight Time)] [info]   Waiting on a storage offer from the network...
 [Tue Sep 27 2016 11:02:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for fa4a3f79cd010f82d4697d5e7c3e00f03881eba9 (retry: 0)
 [Tue Sep 27 2016 11:02:55 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for fa4a3f79cd010f82d4697d5e7c3e00f03881eba9 (retry: 1)
 [Tue Sep 27 2016 11:02:58 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for fa4a3f79cd010f82d4697d5e7c3e00f03881eba9 (retry: 2)
 [Tue Sep 27 2016 11:03:02 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for fa4a3f79cd010f82d4697d5e7c3e00f03881eba9 (retry: 3)
 [Tue Sep 27 2016 11:03:05 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for fa4a3f79cd010f82d4697d5e7c3e00f03881eba9 (retry: 4)
 [Tue Sep 27 2016 11:03:19 GMT+0200 (W. Europe Daylight Time)] [warn]   [ 20GB ] Error occurred. Triggering cleanup...
 [Tue Sep 27 2016 11:03:19 GMT+0200 (W. Europe Daylight Time)] [info]   [ 20GB ] Cleaning up...
fs.js:758
  return binding.rmdir(pathModule._makeLong(path));
                 ^

Error: ENOTEMPTY: directory not empty, rmdir 'D:\temp\storj-5700jSrINUvGlY9D'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at _rmdirRecursiveSync (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:285:10)
    at _cleanupCallback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:409:5)
    at Uploader._cleanup (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:118:3)
    at UploadState.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:275:14)
    at null._onTimeout (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:607:15)
    at Timer.listOnTimeout (timers.js:92:15)

Waiting for 0 seconds, press a key to continue ...

```

### Steps to Reproduce

Upload a 20GB file.

**Comment by MeijeSibbel** [comment]
@aleitner , i have found the source of this error and i can now reproduce it every time.

How to reproduce:
-Imagine you have a text file that includes at line 2 of the file the command: 'storj -k meije0612 -c 12 upload-file  57f65198c60b6e4632611a61 D:\films\*'
- Then, 3 lines down you hit the space-bar
- Then hit CTRL+A to copy everything
-It will now copy everything from the beginning of storj-command up to the where you hit the space-bar.
- So basically you are copying 3 line of which only the first line is actual code.
- Storj chokes on this because indeed the path-String if way to long, causing it to crash.

So a filter has to be build-in to ignore everything after the actual upload command. I hope this helps!.

**Comment by MeijeSibbel** [comment]
Correction, it appears i am getting this now every single time i try to upload a file, even when i do not do as described above, i am connected to bridge: http://server019.storj.dk

```
C:\Users\Dani Marion>storj -k meije0612 -c 12 upload-file 57f65198c60b6e4632611a61 D:\films\*
 [Sat Oct 08 2016 12:31:22 GMT+0200 (W. Europe Daylight Time)] [warn]   A redundancy of 0 means files will not be mirrored!
 [Sat Oct 08 2016 12:31:22 GMT+0200 (W. Europe Daylight Time)] [info]   30 file(s) to upload.
 [Sat Oct 08 2016 12:31:23 GMT+0200 (W. Europe Daylight Time)] [info]   Generating encryption key...
 [Sat Oct 08 2016 12:31:23 GMT+0200 (W. Europe Daylight Time)] [info]   Encrypting file "D:/films/A.Promise.2013.720p.BluRay.x264.YIFY.mp4"
 [Sat Oct 08 2016 12:32:08 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Encryption complete
 [Sat Oct 08 2016 12:32:08 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Creating storage token.
.. (retry: 0)
 [Sat Oct 08 2016 12:32:09 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Storing file, hang tigh
t!
 [Sat Oct 08 2016 12:32:09 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\bded3297829e index 0
 [Sat Oct 08 2016 12:32:10 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\81c3bbac75f1 index 1
 [Sat Oct 08 2016 12:32:10 GMT+0200 (W. Europe Daylight Time)] [info]   Hash for this shard is: b289fd43f6d48cf6da42734532bc0e4a347eaff3
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Audit generation for shard done.
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Waiting on a storage offer from the network...
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 0)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\fba8f3780f22 index 2
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Hash for this shard is: 14b0c13501b90d5a920419829c7e8fed21e66bfa
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 1)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 2)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 3)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 4)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 5)
 [Sat Oct 08 2016 12:32:14 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 6)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 7)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 8)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 9)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 10)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 11)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 12)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 13)
 [Sat Oct 08 2016 12:32:15 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 14)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 15)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 16)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 17)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 18)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 19)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 20)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 21)
 [Sat Oct 08 2016 12:32:16 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 22)
 [Sat Oct 08 2016 12:32:17 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 23)
 [Sat Oct 08 2016 12:32:17 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for b289fd43f6d48cf6da42734532bc0e4a347
eaff3 (retry: 24)
 [Sat Oct 08 2016 12:32:17 GMT+0200 (W. Europe Daylight Time)] [warn]   Failed to upload shard...
 [Sat Oct 08 2016 12:32:17 GMT+0200 (W. Europe Daylight Time)] [warn]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Error occurred. Trigger
ing cleanup...
 [Sat Oct 08 2016 12:32:17 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Cleaning up...
fs.js:758
  return binding.rmdir(pathModule._makeLong(path));
                 ^

Error: ENOTEMPTY: directory not empty, rmdir 'E:\Upload\temp\storj-4292qYBkMZOp6nyC'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at _rmdirRecursiveSync (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:2
85:10)
    at _cleanupCallback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:409:
5)
    at Uploader._cleanup (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:120:3)
    at Uploader._handleFailure (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:3
60:10)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:407:10
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\async\lib\async.js:52:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\async\lib\async.js:1209:30
    at UploadState.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:277:
9)
    at _handleError (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:506:18)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:540:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:352:16
    at Request._callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:1013
:14)
    at Request.self.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\node_modules\request\reques
t.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)

C:\Users\Dani Marion>
```

**Comment by MeijeSibbel** [comment]
This error also occurs in core 4.0.1:

```
C:\Users\Dani Marion\Desktop>(
echo 1
 storj --url http://apipool001.storj.dk -k meije0612 upload-file 57f8cc5db6422c4ee767ee6a D:\films\*
 TIMEOUT 2
)
1
 [Tue Oct 11 2016 17:48:55 GMT+0200 (W. Europe Daylight Time)] [warn]   A redundancy of 0 means files will not be mirrored!
 [Tue Oct 11 2016 17:48:55 GMT+0200 (W. Europe Daylight Time)] [info]   30 file(s) to upload.
 [Tue Oct 11 2016 17:48:57 GMT+0200 (W. Europe Daylight Time)] [info]   Generating encryption key...
 [Tue Oct 11 2016 17:48:57 GMT+0200 (W. Europe Daylight Time)] [info]   Encrypting file "D:/films/A.Promise.2013.720p.BluRay.x264.YIFY.mp4"
 [Tue Oct 11 2016 17:49:44 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Encryption complete
 [Tue Oct 11 2016 17:49:44 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Creating storage token... (retry: 0)
 [Tue Oct 11 2016 17:49:45 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Storing file, hang tight!
 [Tue Oct 11 2016 17:49:45 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\4d8ab2c514cc index 0
 [Tue Oct 11 2016 17:49:46 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\f258446ee6e9 index 1
 [Tue Oct 11 2016 17:49:46 GMT+0200 (W. Europe Daylight Time)] [info]   Hash for this shard is: 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Audit generation for shard done.
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Waiting on a storage offer from the network...
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 0)
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 1)
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Trying to upload shard E:\Upload\temp\7aec2feea33c index 2
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Hash for this shard is: a549eccd84bfa7b42f6c7b172a66aeb13b480ceb
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 2)
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 3)
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 4)
 [Tue Oct 11 2016 17:49:50 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 5)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 6)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 7)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 8)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 9)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 10)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 11)
 [Tue Oct 11 2016 17:49:51 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 12)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 13)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 14)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 15)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 16)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 17)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 18)
 [Tue Oct 11 2016 17:49:52 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 19)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 20)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 21)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 22)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 23)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   Querying bridge for contract for 1f6db55c0a871dfb9c2a757b51c3b3e0ba20d8f5 (retry: 24)
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [warn]   Failed to upload shard...
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [warn]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Error occurred. Triggering cleanup...
 [Tue Oct 11 2016 17:49:53 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Cleaning up...
fs.js:758
  return binding.rmdir(pathModule._makeLong(path));
                 ^

Error: ENOTEMPTY: directory not empty, rmdir 'E:\Upload\temp\storj-3240N1ln57aTNIug'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at _rmdirRecursiveSync (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:285:10)
    at _cleanupCallback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:409:5)
    at Uploader._cleanup (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\bin\actions\uploader.js:120:3)
    at Uploader._handleFailure (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\bin\actions\uploader.js:358:10)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\bin\actions\uploader.js:405:10
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\async\lib\async.js:52:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\async\lib\async.js:1209:30
    at UploadState.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\bin\actions\uploader.js:277:9)
    at _handleError (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\storj-lib\lib\bridge-client\index.js:506:18)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\storj-lib\lib\bridge-client\index.js:540:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\storj-lib\lib\bridge-client\index.js:352:16
    at Request._callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\storj-lib\lib\bridge-client\index.js:1013:14)
    at Request.self.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj-cli\node_modules\storj-lib\node_modules\request\request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)

Waiting for 0 seconds, press a key to continue ...
````

**Comment by MeijeSibbel** [comment]
Problem is still occurring in the latest core release:

``` 

[Mon Oct 17 2016 00:05:20 GMT+0200 (W. Europe Daylight Time)] [warn]   Failed to upload shard...
 [Mon Oct 17 2016 00:05:21 GMT+0200 (W. Europe Daylight Time)] [warn]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Error occurred. Triggering cleanup...
 [Mon Oct 17 2016 00:05:21 GMT+0200 (W. Europe Daylight Time)] [info]   [ A.Promise.2013.720p.BluRay.x264.YIFY.mp4 ] Cleaning up...
fs.js:758
  return binding.rmdir(pathModule._makeLong(path));
                 ^

Error: ENOTEMPTY: directory not empty, rmdir 'E:\Upload\temp\storj-6828evVCCpTHucWV'
    at Error (native)
    at Object.fs.rmdirSync (fs.js:758:18)
    at _rmdirRecursiveSync (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:285:10)
    at _cleanupCallback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\tmp\lib\tmp.js:409:5)
    at Uploader._cleanup (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:125:3)
    at Uploader._handleFailure (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:358:10)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:405:10
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\async\lib\async.js:52:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\node_modules\async\lib\async.js:1209:30
    at UploadState.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-cli\bin\actions\uploader.js:284:9)
    at _handleError (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:506:18)
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:540:16
    at C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:352:16
    at Request._callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\lib\bridge-client\index.js:1009:14)
    at self.callback (C:\Users\Dani Marion\AppData\Roaming\npm\node_modules\storj\node_modules\storj-lib\node_modules\request\request.js:187:22)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)

Waiting for 0 seconds, press a key to continue ...

C:\Users\Dani Marion\Desktop>(
echo 557
 storj -k meije0612 -r 12 upload-file 57af3a1e8f266a7d4e391055 D:\films\*
 TIMEOUT 2
)
557

```
